charts/rancher-istio/1.5.700/templates/istio-onefive-migration.yaml

```diff
@@ -0,0 +1,77 @@
+---
```

Comment:
This is the migration job

Reply:
Will the migration job be always running no matter if it is fresh install or upgrade? Is it better to have an answer `migration: enabled` to control this?

Reply:
That's probably a good idea just to disable it in case its causing problems. But in reality it would alway run because we never know if 1.4 exists at upgrade time.

Reply:
Added enabled flag

Reply:
We can add it and it doesn't hurt. When this is no longer needed the flag can always be false. :+1: